deploy/docker/setupjob/Makefile

```diff
@@ -0,0 +1,20 @@
+BUILD_TAG ?= latest
+IMAGE_NAME = setupjob
```

Comment:
This should rather be a full name like: `sumologic-kubernetes-setup` or sth.

Reply:
Renamed it to `kubernetes-setup` as the alias we're going to get will be `sumologic` (hopefully :) )